app/controllers/jobs_controller.rb

```diff
@@ -1,17 +1,35 @@
 class JobsController < ApplicationController
+
   def index
+    if params[:sort]
+      @company = Company.find(params[:company_id])
+      @jobs = Job.where(city: params[:sort], company: @company)
+      @contact = Contact.new
+      @contacts = Contact.all
```

Comment:
do we want to see all contacts here or just contacts associated with this company